content/v2.1/helpers/form_helper.md

```diff
@@ -0,0 +1,964 @@
+---
+title: Form Helper
+order: 30
+---
+# Form Helper
+
+Yields a form builder for constructing an HTML form and returns the resulting form string.
+
+Please read this section for the methods for building the form's fields.
+
+## form_for
+
+Builds the form using the given base name for all fields.
+The overall structure of the attributes available is as follows:
+
+### Usage
+
+```ruby
```

Comment:
@swilgosz As a first-time reader, this is overwhelming.
Can we skip or move to advanced usage of the helper?

Reply:
Done. Removed for now.